dhis-2/dhis-web-api-test/src/test/java/org/hisp/dhis/servlet/filter/RequestIdentifierFilterTest.java

```diff
@@ -88,23 +87,24 @@ void testIsEnabled()
         IOException
     {
         init( true );
-        doFilter();
+        doFilter( request -> {
+            HttpSession session = mock( HttpSession.class );
+            when( request.getSession() ).thenReturn( session );
+            when( session.getId() ).thenReturn( "ABCDEFGHILMNO" );
+        } );
 
         assertNotNull( MDC.get( "sessionId" ) );
```

Comment:
would it also make sense that we put the session id `ABCDEFGHILMNO` into the context?

Reply:
Not sure what you suggest. Please elaborate.

Reply:
I don't know the code we are testing. I just based it on the class name and that we are asserting on MDC. I assume what we are testing is responsible for putting the session id into the MDC (context) that is then used to log the session id.

https://www.baeldung.com/mdc-in-log4j-2-logback

That's why I thought should we not also check that the value of `MDC.get( "sessionId" )` is the session id `ABCDEFGHILMNO`

Reply:
Ah, yes - that makes sense to me.

Reply:
Added the change to #11997 as that so far had just comment changes.

Reply:
Thank you 😄